components/CancelButton/CancelButton.styled.js

```diff
@@ -0,0 +1,14 @@
+import { styled } from "styled-components";
+import Link from "next/link";
+
+export const StyledCancelButton = styled(Link)`
+  background-color: red;
+  writing-mode: horizontal-tb !important;
+  margin: 0em;
+  padding-inline: 6px;
+  border-style: outset;
+  border-color: buttonborder;
+  font-size: 13px;
+  text-decoration: none;
+  color: white;
```

Comment:
It's best to use color codes (e.g. hex codes, rgb, hsl) instead of color names in production.
```suggestion
  color: #ffffff
```